specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimproducts.json

```diff
@@ -914,7 +918,282 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/products/{productId}/tags": {
+      "get": {
+        "tags": [
+          "ProductTags"
+        ],
+        "operationId": "Tags_ListByProduct",
```

Comment:
This operation is `Tags_ListByProduct` the next is `Tag_HeadByProduct`

Reply:
We have used a singular case in all other operations. @vfedonkin let's use Tag.

Reply:
Done, using "Tag" everywhere.